sample_factory/algorithms/appo/learner.py

```diff
@@ -332,7 +332,7 @@ def _calculate_gae(self, buffer):
 
     def _mark_rollout_buffer_free(self, rollout):
         r = rollout
-        self.traj_tensors_available[r.worker_idx, r.split_idx][r.env_idx, r.agent_idx, r.traj_buffer_idx] = 1
+        self.traj_tensors_available[r.worker_idx, r.split_idx, r.env_idx, r.agent_idx, r.traj_buffer_idx] = 1
```

Comment:
I am assuming this also goes away when we use queues for everything, right?

Reply:
Yeah, it will when using a queue.  I do think that this logic should be kept tho as the static assignment makes sense when there are only a handful of environments per actor worker.